client/src/components/DataFiles/DataFiles.js

```diff
@@ -143,10 +143,14 @@ const DataFiles = () => {
     );
   }
 
+  if (listingParams.system === 'a2cps.storage.community') {
```

Comment:
Can we have a settings config set for this similar to how we handled the Work2 message, rather than a string? Would allow us a bit more flexibility if the system id changed.

Reply:
@fnets here's how we did it for work2
https://github.com/TACC/Core-Portal/pull/373

Reply:
Thanks! I think I got it!